bitcoin/src/util/mod.rs

```diff
@@ -17,8 +17,6 @@ pub mod psbt;
 pub mod taproot;
 pub mod sighash;
 
-pub(crate) mod endian;
```

Comment:
I don't see the **file** removed, looks like you forgot?

Reply:
Lol, great catch. It'd probably be years otherwise before we noticed.

Reply:
I've always thought it odd that clippy does not warn for unused files in a source tree.